### PR TITLE
Feat/emoji picker

### LINKED ISSUE #31: Emoji picker
Emoji select input for the emoji field on the basic profile on /profile/edit-profile

### PR DESCRIPTION
**Pull request by Dhaiwat10**
Closes #31 

I've targeted the PR at the staging branch. Let me know if this should be main instead.

### CODE REVIEW

**Comment by Dhaiwat10** [comment]
I think there are three extra commits that are popping up because I created my new branch off of the `main` branch..sorry about that. I think we'll have to rebase/squash

File: packages/react-app/pages/profile/edit-profile.js [hunk 1]
```diff
@@ -9,6 +19,8 @@ import modelAliases from "../../model.json";
 import { ceramicCoreFactory, CERAMIC_TESTNET, CERAMIC_TESTNET_NODE_URL } from "../../ceramic";
 import { Web3Context } from "../../helpers/Web3Context";
 
+const emojis = ["🚀", "🙏", "👻", "🤌", "👀"];
```
Comment (Cali93) [inline packages/react-app/pages/profile/edit-profile.js:22]: Can you use this array of emoji instead ? https://gist.github.com/anthonydelgado/528d1fab9242067348c0ac25f873d7f0

**Review by Cali93 — CHANGES_REQUESTED** [review]

**Comment by Dhaiwat10** [comment]
Done

**Comment by QEDK** [comment]
@Dhaiwat10 can you address the conflicts, ty

**Comment by Dhaiwat10** [comment]
@QEDK done.

**Review by Cali93 — APPROVED** [review]

**Pull request merged**